### DISCUSSION TITLE
How about adding custom keybinds

### DISCUSSION THREAD

**Discussion by aDubbzAU** [FunkinCrew/Funkin#1775] [Ideas]
I've seen it in some mods, and to me DFJK is better than WASD (or I guess it's better to say ASWD here) or arrows
Example: in normal FNF I struggle with South on normal, but with the Big Brother Mod (it includes the entire base game 'cause it adds the F**k... difficulty) I set my keybinds to DFJK and managed to complete week 2 on HARD. (BTW, if anyone has tried the Big Brother mod, but deleted it because it crashed on week 1: Dadbattle is broken for whatever reason, the rest works. Also that exact mod doesn't render/ doesn't use sustain notes, only the start note for sustain notes appears (again, that part is for a SPECIFIC mod))
Anyway...
Custom keybinds?

**Comment by CubixL** [comment]
the week 7 newgrounds release already has an option for those.
it's just not out for itch.io yet
